met-web/src/components/engagement/view/EngagementWidget.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { MetPaper } from 'components/common';
+import { Grid, Typography, Avatar, Link } from '@mui/material';
+
+const testData = {
+    image: 'https://cdn.pixabay.com/photo/2017/09/25/13/12/cocker-spaniel-2785074__340.jpg',
+    name: 'Remy Sharp',
+    email: 'TheJad@bc.gov.ca',
+    phone: '647-543-1332',
+};
+
+export default function EngagementWidget() {
+    return (
+        <Grid container alignItems="center" justifyContent="center" spacing={2} xs={12}>
+            <Grid item xs={11}>
+                <MetPaper elevation={1} sx={{ padding: '1em', minHeight: '10em' }}>
+                    <Grid container item justifyContent="flex-start" alignItems="center" xs={12}>
```

Comment:
A grid container contains Grid items, if it doesn't have grid items, it doesn't need to be a container